foundation/foundation-mda/src/main/resources/templates/cdi.container.factory.java.vm

```diff
@@ -57,11 +58,14 @@ public final class CdiContainerFactory {
     protected static List<CdiContext> getContexts() {
         List<CdiContext> contexts = new ArrayList<>();
         contexts.add(new PipelinesCdiContext());
+        ## TODO: would be nice to only include if we need messaging, but right now we assume messaging everywhere
+        contexts.add(new MessagingCdiContext());
```

Comment:
I: We were already using composition of contexts here for alerting and data lineage, so also included messaging to simplify the pipeline CDI context base class.  In that class we were already assuming messaging was required, so don't intend to resolve this todo right now.